src/lightning/fabric/fabric.py

```diff
@@ -476,7 +476,9 @@ def to_device(self, obj: Union[nn.Module, Tensor, Any]) -> Union[nn.Module, Tens
         return move_data_to_device(obj, device=self.device)
 
     def print(self, *args: Any, **kwargs: Any) -> None:
-        """Print something only on the first process.
+        """Print the arguments to standard output. When running on multiple devices on a single node, this method
+        will only print from one process to avoid redundant outputs. When running across multiple nodes, this
+        method will print from one process in each node.
```

Comment:
This is overly verbose in my opinion, considering the trivial functionality this method provides. Let's make sure users can easily understand it in a single sentence?

```suggestion
        """Print something only on the first process. If running on multiple machines, it will print once per machine.
```

Reply:
I don't have a preference. Since you didn't approve it I'll assume that this is not a nit so I will update it with your suggestion here and in the Trainer

Reply:
Yeah I think it could be considered a nit, maybe I'm focusing too much on this and it's a non issue 🤷